Vagrantfile

```diff
@@ -31,7 +39,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     v.vm.box_url = "http://files.vagrantup.com/precise64_vmware.box"
   end
 
-  config.vm.synced_folder ".", "/srv", owner: 'www-data', group: 'www-data', mount_options: ["dmode=775", "fmode=664"]
+  if (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) == nil
+    config.vm.synced_folder ".", "/srv", nfs: true
```

Comment:
Can we remove this part? I don't really want to do NFS right now because it breaks some things like permissions.